volontulo/utils.py

```diff
@@ -41,3 +41,14 @@ def save_history(req, obj, action):
         object_repr=str(obj),
         action_flag=action
     )
+
+
+def offer_statuses():
```

Comment:
Why is it callable? Make simple variable available from this module (in future maybe separate const.py?)